Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -156,7 +143,10 @@ public virtual void OnValidate()
             // add transport if there is none yet. makes upgrading easier.
             if (transport == null)
             {
-                gameObject.AddComponent<TelepathyTransport>();
+                transport = gameObject.AddComponent<TelepathyTransport>();
```

Comment:
should probably check for existing getcomponent<itransport> first and use that. otherwise addcomponent

Reply:
yes,  that is precisely why I put work-in-progress label.

Reply:
By the way,  all my pull requests are editable by you,  so if you feel like adding that code,  go right ahead.  I would not be able to get to it until the evening